enable/label.py

```diff
@@ -191,25 +191,6 @@ def _draw_mainlayer(self, gc, view_bounds=None, mode="normal"):
         with gc:
             gc.translate_ctm(*self.position)
 
-            # Draw border and fill background
```

Comment:
This was removed because it was drawing the border twice; and this code was drawing it in the wrong position.

Reply:
I gather than `Component` is in charge of drawing the border?